ccf_transaction_processor/scripts/configure_ccf_network.py

```diff
@@ -34,7 +35,7 @@
 from loguru import logger as LOG
 
 ## -----------------------------------------------------------------
-ContractHome = os.environ.get("PDO_HOME") or os.path.realpath("/opt/pdo")
+ContractHome = os.environ.get("CCF_HOME") or os.path.realpath("/opt/pdo")
 CCF_Etc = os.path.join(ContractHome, "ccf", "etc")
```

Comment:
ditto.

Reply:
same answer as above

Reply:
FYI... this one is different than the others. You're using PDO_HOME (preferred) instead of PDO_INSTALL_ROOT.